### PR TITLE
Add minimum required PR approval app to showcase

### PR DESCRIPTION
**Pull request by Phanatic**
👋 
Working with the awesome students at [Naperville North High School](https://www.naperville203.org/nnhs) we've created this bot as part of an initiative to get high school students working with Industry Professionals. We'd love to have this app be showcased in the Probot apps showcase page.

This GitHub app allows a repository maintainer to require that more than one Reviewer with write access to a repository needs to approve a PR before it can be merged.

### CODE REVIEW

File: _apps/prr-review-counter.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+slug: pull-request-review-counter
+# Include a few screenshots that show your app in action
+screenshots:
+- https://user-images.githubusercontent.com/82819/37225532-c290047e-238b-11e8-9443-0322d16cc0b1.png
+# The GitHub usernames of anyone who authored the app
+authors: [ gcschmit, rycao18, saanvid, helenagao14 ]
+# The repository where the code is located
+repository: probotics2018/PRRCounter
+# The address where this app is deployed
+host: https://github.com/apps/pull-request-review-counter
```
Comment (bkeepers) [inline _apps/prr-review-counter.md]: This should be the URL of the deployed app on heroku, glitch, now, or etc

File: _apps/prr-review-counter.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+# A human-friendly name of your listing
+title: Pull Request Review Counter
+# A short description of what your app does
+description: Require multiple approved reviews before merging a Pull Request.
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: pull-request-review-counter
+# Include a few screenshots that show your app in action
+screenshots:
+- https://user-images.githubusercontent.com/82819/37225532-c290047e-238b-11e8-9443-0322d16cc0b1.png
```
Comment (bkeepers) [inline _apps/prr-review-counter.md]: Could you zoom this out a bit so it includes the whole checks box? Similar to https://probot.github.io/apps/dco/

It'd also be great to have a second screenshot of it failing.

File: _apps/prr-review-counter.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# The GitHub usernames of anyone who authored the app
+authors: [ gcschmit, rycao18, saanvid, helenagao14 ]
+# The repository where the code is located
+repository: probotics2018/PRRCounter
+# The address where this app is deployed
+host: https://github.com/apps/pull-request-review-counter
+---
+
+Pull Request Review Counter ensures that all PRs in your repository can only be merged after X number of reviewers with write access have approved the PR.
```
Comment (bkeepers) [inline _apps/prr-review-counter.md]: How is the app configured? It'd be great to include docs for that here.

File: _apps/prr-review-counter.md (hunk 1 above)
Comment (rycao18) [reply _apps/prr-review-counter.md]: @bkeepers Sorry about that, here you go
https://hidden-caverns-26207.herokuapp.com/

File: _apps/prr-review-counter.md (hunk 3 above)
Comment (rycao18) [reply _apps/prr-review-counter.md]: @Phanatic @bkeepers The app relies on a file being in the master branch of the repository, which should be called `Github.json` , which should be structured like this 

`{
	"approved-reviewer-count": 0
}
`

File: _apps/prr-review-counter.md (hunk 2 above)
Comment (rycao18) [reply _apps/prr-review-counter.md]: phani1
:)

**Comment by dessant** [comment]
It seems this feature is now natively supported by GitHub.

https://blog.github.com/2018-03-23-require-multiple-reviewers/

**Pull request closed without merging**